src/test/java/guitests/InvalidLoginTest.java

```diff
@@ -32,9 +31,8 @@ public void invalidLoginTest() throws InterruptedException {
 
     @Test
     public void emptyLoginTest() throws InterruptedException {
-        TextField repoOwnerField = find("#repoOwnerField");
-        doubleClick(repoOwnerField);
-        doubleClick(repoOwnerField);
+        find("#repoOwnerField");
+        selectAll();
```

Comment:
I think they should be kept separate. One reason would be readability.
Another is that sometimes only find is used (to check the text) or only
selectAll is used (to change, say, default repo or filter again and again).
On Nov 8, 2015 6:31 PM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In src/test/java/guitests/InvalidLoginTest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1123#discussion_r44223332>:
>
> > @@ -32,9 +31,8 @@ public void invalidLoginTest() throws InterruptedException {
> >
> >      @Test
> >      public void emptyLoginTest() throws InterruptedException {
> > -        TextField repoOwnerField = find("#repoOwnerField");
> > -        doubleClick(repoOwnerField);
> > -        doubleClick(repoOwnerField);
> > +        find("#repoOwnerField");
> > +        selectAll();
>
> If these two lines always appear together, may be they should be combined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1123/files#r44223332>.
>